sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Storage/BlobsCheckpointStore.cs

```diff
@@ -21,7 +22,7 @@ internal sealed class BlobsCheckpointStore : PartitionManager
         private static readonly Regex s_doubleQuotesExpression = new Regex("\"(.*)\"", RegexOptions.Compiled);
 
         /// <summary>The client used to interact with the Azure Blob Storage service.</summary>
-        private readonly BlobContainerClient _containerClient;
+        private readonly BlobContainerClient ContainerClient;
```

Comment:
Mind if we make this a property?   We've been doing that in many of the other places for readonly instance members to avoid people disagreeing on the `_` need.

Reply:
I'll do it.